src/components/Person/_person.scss

```diff
@@ -0,0 +1,116 @@
+// ----------------------------------
+// UI Components
+// Person
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+:root {
+  --person-image-size-small: calc(10 * var(--space-unit)); // 80px
+  --person-image-size-medium: 162px; // temporary
+}
+
+.cc-person {
+  margin-bottom: var(--space-lg);
+
+  @include mq(sm) {
+    display: flex;
+  }
+
+  a {
+    @include animated-link;
+  }
+}
+
+.cc-person__image {
```

Comment:
There's an issue with the image width being squashed > 768px. Applying `flex-shrink: 0;` will force the image width to remain constant. Also by default the figure element this selector is applied to looks like it could use some margin resetting. The excess left hand margin is very noticeable on mobile.

![Screenshot 2021-08-03 at 17 29 21](https://user-images.githubusercontent.com/49439125/128053221-a4cc6abe-b83b-4fa4-974f-08d0fd383b44.png)